operator/cmd/run/run.go

```diff
@@ -239,19 +273,119 @@ func Run(
 	forceDefluxedMode bool,
 	pprofAddr string,
 	helmRepositoryURL string,
+	webhookCertPath string,
```

Comment:
This method is painfully large at this point. Would you mind following up with a small refactor to instead build the `webhook.Server` and metrics options elsewhere (In the closure or in dedicated helpers) and just passing those in as arguments to make it slightly less monolithic?

Reply:
Sure I can follow up with next PR